test/rules/_integration/enable-disable/test.ts.lint

```diff
@@ -78,9 +78,6 @@ var AAAaA = 'test' // previously both `quotemark` and `variable-name` rules were
 var AAAaA = 'test' // tslint:disable-line:quotemark
 var AAAaA = 'test' // ensure that disable-line rule correctly handles previous `enable rule - disable all` switches
 
-// tslint:enable:no-var-keyword
```

Comment:
What I'm actually trying to say: You disable a rule by setting `severity: off`. Now you turn it on via comment. What severity should this rule have? How do I set a specific severity other than the default?